pkg/gotohelm/helmette/sprig.go

```diff
@@ -12,8 +12,19 @@ import (
 
 	"github.com/imdario/mergo"
 	"golang.org/x/exp/maps"
+	"gopkg.in/yaml.v3"
 )
 
+// ToYAML is the go equivalent of sprig's `toYaml`.
```

Comment:
super nit: apparently `toYaml` is from helm not sprig. Sprig only provides toJSON.